frontend/src/metabase/query_builder/components/filters/modals/InlineDatePicker/InlineDatePicker.styled.ts

```diff
@@ -0,0 +1,50 @@
+import styled from "@emotion/styled";
+import { space } from "metabase/styled-components/theme";
+import { color, alpha } from "metabase/lib/colors";
+
+import Button from "metabase/core/components/Button";
+
+export const OptionContainer = styled.div`
+  grid-column: span 2 / span 2;
+  margin: ${space(2)} 0;
+  padding-bottom: ${space(2)};
+  font-weight: bold;
+  border-bottom: 1px solid ${color("border")};
+`;
+
+type OptionButtonProps = {
+  primaryColor?: string;
```

Comment:
I don't see you passing `primaryColor` to `<OptionButton>`, only `selected`. Am I missing something? Because if you can remove this prop then we could simplify the use of this prop in each emotion component quite a bit.